Plugins/Sycl/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# This module needs at least CMake 3.13.
+cmake_minimum_required( VERSION 3.13 )
```

Comment:
I do not think this is needed since it is already part of the `FindSYCL` module.